File: src/main/java/akeefer/model/BenachrichtigunsIntervall.java
package akeefer.model;

public enum BenachrichtigunsIntervall {

    deaktiviert(0, null),
    taeglich(1, "gestern"),
    woechnetlich(7, "letzte Woche");

    private int tage;
    private String zeitinterval;

    BenachrichtigunsIntervall(int tage, String zeitinterval) {
        this.tage = tage;
        this.zeitinterval = zeitinterval;
    }

    public int getTage() {
        return tage;
    }

    public String getZeitinterval() {
        // TODO (ak) lokalisieren
        return zeitinterval;
    }
}

File: backend/src/main/kotlin/com/github/xtermi2/virtualrun/model/BenachrichtigunsIntervall.kt
package com.github.xtermi2.virtualrun.model

enum class BenachrichtigunsIntervall(val tage: Int,
                                     val zeitinterval: String?) {
    deaktiviert(0, null),
    taeglich(1, "gestern"),
    woechnetlich(7, "letzte Woche");
}